packages/keyring-controller/package.json

```diff
@@ -48,6 +48,7 @@
     "@metamask/scure-bip39": "^2.1.0",
     "@types/jest": "^27.4.1",
     "deepmerge": "^4.2.2",
+    "immer": "^9.0.6",
```

Comment:
It seems that the `Patch` type from `immer` is being used in a type that is exported, so this should probably be a production dependency